test/test_server.py

```diff
@@ -66,13 +66,13 @@ def setUp(self):
     def tearDown(self):
         if self.skip: return
 
-        if self.p.poll() == None:
+        if self.p.poll() is None:
```

Comment:
This PR includes above changes about None comparison.
I think it would be better to use 'is' not '=='.

Reply:
While you are right (one should use `is` for `None` checks) in this case it does not matter much. The current implementation is not wrong.